### ISSUE TITLE
[Horizon] Modeline active and inactive foreground should be swapped

### ISSUE THREAD

**Issue by ianpan870102** [hlissner/emacs-doom-themes#401]
In doom-horizon (thx for the port btw!) the foreground face of modeline under active state and inactive state should be swapped like so:

```
    (modeline-fg     (doom-darken fg 0.2))
    (modeline-fg-alt (doom-lighten bg 0.2))
```

**Comment by ianpan870102** [comment]
I just created a PR: https://github.com/hlissner/emacs-doom-themes/pull/402

**Comment by ema2159** [comment]
Fixed on #402!
